docker-entrypoint.sh

```diff
@@ -44,21 +44,11 @@ if [ "$1" = 'postgres' ]; then
 
 		{ echo; echo "host all all 0.0.0.0/0 $authMethod"; } >> "$PGDATA/pg_hba.conf"
 
-		set_listen_addresses '' # we're going to start up postgres, but it's not ready for use yet (this is initialization), so don't listen to the outside world yet
-
-		gosu postgres "$@" &
-		pid="$!"
-		for i in {30..0}; do
-			if echo 'SELECT 1' | psql --username postgres &> /dev/null; then
```

Comment:
Maybe we should keep this or a similar test to make sure it is ready to run some SQL?